modules/standard/BigInteger.chpl

```diff
@@ -308,7 +308,7 @@ module BigInteger {
     pragma "last resort"
     @deprecated("the argument name 'str' is deprecated - please use 'x' instead")
     proc init(str: string, base: int = 0) throws where bigintInitThrows == true
-      do this.init(num);
+      do this.init(str);
```

Comment:
I believe this needs to be `init(str, base)` for true correctness

Reply:
Makes sense, thanks